frankenwm.c

```diff
@@ -28,15 +29,6 @@
 #endif
 
 /* upstream compatility */
-#define True  true
-#define False false
-#define Mod1Mask     XCB_MOD_MASK_1
-#define Mod4Mask     XCB_MOD_MASK_4
-#define ShiftMask    XCB_MOD_MASK_SHIFT
-#define ControlMask  XCB_MOD_MASK_CONTROL
-#define Button1      XCB_BUTTON_INDEX_1
-#define Button2      XCB_BUTTON_INDEX_2
-#define Button3      XCB_BUTTON_INDEX_3
```

Comment:
What's up with these?

Reply:
I was getting compiler warnings about re-defining things.
This was Maya's suggestion.
If I understood it correctly, these are already defined in Xlib.h, correct @coypoop

Reply:
Yes, I was trying to avoid compiler warnings.